src/compas_view2/shapes/collection.py

```diff
@@ -0,0 +1,24 @@
+from compas.datastructures import Mesh
+
+
+__all__ = ['Collection']
+
+
+class Collection(Mesh):
```

Comment:
still looks weird. would it not make more sense that it inherits from the buffer object directly?